lib/src/main/java/org/exoplatform/social/addons/updater/IdentityMigrationService.java

```diff
@@ -284,15 +284,17 @@ public void doRemove() throws Exception {
 
   private Identity migrateIdentity(Node node, String jcrId) throws Exception {
     String providerId = node.getProperty("soc:providerId").getString();
-    String remoteId = node.getProperty("soc:remoteId").getString();
+    // The node name is the identity id.
+    // Node name is soc:<name>, only the <name> is relevant
+    String name = node.getName().substring(4);
```

Comment:
The node name is encoded for example node.name => node%02name so a substring is not enough